### ISSUE TITLE
[ALL] published 3.x branch of swagger-codegen-cli.jar might be broken

### ISSUE THREAD

**Issue by cztomsik** [swagger-api/swagger-codegen#11094]
##### Description
For some reason the result from the source code was not reflecting the output in the console so that was really weird so I got digging and apparently, published jar file `https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.27/swagger-codegen-cli-3.0.27.jar` contains two classes named `TypeScriptAngularClientCodegen` (with **different code**) and it also contains two different mustache templates.

I have no idea where the different version got from because I can't find it in this repository but it's definitely the one which is being executed for me (and it's missing lots of newer updates)

**The 2.x branch seems to be ok.**

##### Swagger-codegen version
3.0.27 (but .26 is broken too and probably few others too)

##### Steps to reproduce
Open published jar in java decompiler, look for `TypeScriptAngularClientCodegen` (Open Type) and I think that there is duplicate type (two classes with the same name) and/or just `jar -xvf swagger-cli.jar` and then do `find . | grep angular` and you should see two duplicate angular templates with slightly different paths.

##### Suggest a fix/enhancement
No idea, I am not doing Java anymore for a quite a few years so I can't really help here but I think it might be just packaging thing.

**Comment by cztomsik** [comment]
Ok, so it looks like the `swagger-codegen-generators` file at `https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-generators/1.0.27/swagger-codegen-generators-1.0.27.jar` is correct, so this is only related to the **CLI build**

**Comment by Fylipp** [comment]
Angular 10 clients don't compile, the TypeScript version does not work with the latest Angular 10 release and compiling it results in this type error:

```
Compiling TypeScript sources through ngc
ERROR: api.module.ts:18:71 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).

18     public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders {
                                                                         ~~~~~~~~~~~~~~~~~~~

```

**Comment by cztomsik** [comment]
yes, they don't compile in 3.x branch but they do in 2.x one (there's one commit which was not ported to 3.x)
or you can just switch to https://github.com/OpenAPITools/openapi-generator which seems to work better in overall
